sdl_android/src/main/java/com/smartdevicelink/managers/BaseSubManager.java

```diff
@@ -53,7 +53,7 @@ protected void transitionToState(int state) {
 		synchronized (STATE_LOCK) {
 			this.state = state;
 		}
-		if(state == READY && completionListener != null){
+		if( (state == READY || state == LIMITED) && completionListener != null ){
 			completionListener.onComplete(true);
 			completionListener = null;
 		}else if(state == ERROR && completionListener != null){
```

Comment:
Can we match the logic between the `start()` method and this one? I think the `start()` method logic is cleaner.

Reply:
Sure. Will do that.